runtime/util/util.cpp

```diff
@@ -34,9 +35,8 @@ block *construct_raw_term(void *subject, char const *sort, bool raw_value) {
   return static_cast<block *>(construct_composite_pattern(tag, args));
 }
 
-void print_proof_hint_header(kllvm::proof_trace_writer *writer) {
+void print_proof_hint_header(proof_trace_writer *writer) {
   uint32_t version = 11;
```

Comment:
You'll probably need to update this version if rebasing doesn't do it automatically for you.

Reply:
Good catch! will do